HISTORY.rst

```diff
@@ -2,9 +2,50 @@
 History
 =======
 
-3.26.0 (2021-01-08)
+3.27.0 (2021-01-21)
 -------------------
 
+Changes:
+
+- Upgrades including Snowfakery 1.4 with 
+  min, max, round functions. PR #2335
```

Comment:
We haven't typically included PR links. I suppose I don't have a big objection to that, but you'll need to do a bit more reformatting to remove them when you post to the Hub and Trailblazer community. For the same reason, I usually turn on the word-wrap in my editor and avoid line breaks within paragraphs, because then they copy over better.

We don't typically mention routine upgrades of dependencies, with the exception of Snowfakery, so I would probably change this to "Updated Snowfakery to version 1.4 with ..."

Reply:
I wouldn't think of them as PR links but as PR numbers and it seems helpful to me across all channels? It allows someone (the technical subset) to investigate further what changed and why?

Reply:
I guess I'm okay with trying this. The audience for CumulusCI is less likely to be confused by it than our product customers.